certified-connectors/DomainToolsIrisEnrich/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1145 @@
+        "account": {
+          "type": "object",
+          "properties": {
+            "api_username": {
+              "type": "string",
+              "description": "API username",
+              "title": "API username"
+            },
+            "active": {
+              "type": "boolean",
+              "description": "Active",
+              "title": "Active"
+            }
+          },
+          "description": "Account Information"
+        },
+        "products": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string",
+                "description": "ID",
+                "title": "ID"
+              },
+              "per_month_limit": {
+                "type": "string",
+                "description": "Per Month Limit",
+                "title": "Per Month Limit"
+              },
+              "per_minute_limit": {
+                "type": "string",
+                "description": "Per Minute Limit",
+                "title": "Per Minute Limit"
+              },
+              "absolute_limit": {
+                "type": "string",
+                "description": "Absolute Limit",
+                "title": "Absolute Limit"
+              },
+              "usage": {
+                "type": "object",
+                "properties": {
+                  "today": {
+                    "type": "string",
+                    "description": "Today Usage",
+                    "title": "Today Usage"
+                  },
+                  "month": {
+                    "type": "string",
+                    "description": "Month Usage",
+                    "title": "Month Usage"
+                  }
+                },
+                "description": "Usage Details"
+              },
+              "expiration_date": {
+                "type": "string",
+                "description": "Expiration Date",
+                "title": "Expiration Date"
+              }
+            }
+          },
+          "description": "Products Information"
+        }
+      },
+      "description": "Acoount Information Response"
+    },
+    "Enrich_Response": {
+      "type": "object",
+      "properties": {
+        "response": {
+          "type": "object",
+          "properties": {
+            "limit_exceeded": {
+              "type": "boolean",
+              "description": "DomainTools API results are limited to 5000 entries in a result set. If a query has more than 5000 results, the 'Limit exceeded' returns true. Try refining your query with more specific search terms.",
+              "title": "Limit Exceeded"
+            },
+            "message": {
+              "type": "string",
+              "description": "message",
+              "title": "Message"
+            },
+            "results_count": {
+              "type": "integer",
+              "format": "int32",
+              "description": "results_count",

```

Comment:
@RamboV All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
Hi @sriyen-msft , Modified the parameter descriptions